task-sdk/src/airflow/sdk/execution_time/task_runner.py

```diff
@@ -443,7 +443,12 @@ def get_message(self) -> ReceiveMsgType:
 
         This will block until the message has been received.
         """
-        line = self.input.readline()
+        line = None
+
+        # We need to investigate why some lines are empty ("")
+        while not line:
+            line = self.input.readline()
+
```

Comment:
Need investigation. Apparently there are some empty lines logged there that need to be skipped.

Reply:
That is slightly worrying, this means something is sending an empty line to the processes stdin, which "shouldn't" happen

Reply:
I added a TODO with a more descriptive text to explain what remain to be done here.